ruby_event_store/spec/client_spec.rb

```diff
@@ -72,6 +72,14 @@ module RubyEventStore
       expect(client.read.stream(stream).to_a).to eq([first_event, second_event])
     end
 
+    specify 'publishing calls after_store_callback if provided' do
+      some_var = 42
+
+      client.publish(TestEvent.new, after_store_callback: ->() { some_var = 13 })
+
+      expect(some_var).to eq(13)
+    end
```

Comment:
This test fails to specify why

```
client.publish(TestEvent.new)
some_var = 13 
expect(some_var).to eq(13)
```

is not a valid solution and why `after_store_callback` needs to be invoked after storing but before calling a broker.